bucket/qq.json

```diff
@@ -0,0 +1,46 @@
+{
+    "version": "9.7.3.28946",
+    "description": "An instant messaging tool that gives you the best way to keep in touch with your friends and family.",
+    "homepage": "https://im.qq.com",
+    "license": {
+        "identifier": "Proprietary",
+        "url": "https://ti.qq.com/agreement/index.html"
+    },
+    "notes": [
+        "Scoop doesn't persist your QQ data, by default they are stored in: ",
+        "'%UserProfile%\\Documents\\Tencent Files'.",
+        "Files produced at runtime leaves in '%AppData%\\Tencent'."
+    ],
+    "url": "https://webcdn.m.qq.com/spcmgr/download/QQ9.7.3.28946.exe#/dl.7z",
+    "hash": "6d11c322a7deb69634de54ba3006f908fa53a09a63585aaf40b02d3dce6fd825",
+    "shortcuts": [
+        [
+            "Bin\\QQScLauncher.exe",
+            "QQ"
+        ]
+    ],
+    "installer": {
+        "script": [
+            "Get-Item \"$dir\\*\" -Exclude .rsrc | Remove-Item",
+            "Expand-7zipArchive \"$dir\\.rsrc\\2052\\MSI\\101\" \"Files\" -Destination \"$dir\\\" -Removal",
+            "Move-Item \"$dir\\Files\\*\" \"$dir\"",
+            "Expand-7zipArchive \"$dir\\.rsrc\\2052\\MSI\\102\" \"vcruntime140.dll\" -Destination \"$dir\\Bin\\\"",
+            "Expand-7zipArchive \"$dir\\.rsrc\\2052\\MSI\\102\" \"msvcp140.dll\" -Destination \"$dir\\Bin\\\"",
+            "Expand-7zipArchive \"$dir\\.rsrc\\2052\\MSI\\102\" \"msvcr100.dll\" -Destination \"$dir\\Bin\\\"",
+            "Expand-7zipArchive \"$dir\\.rsrc\\2052\\MSI\\102\" \"vcomp140.dll\" -Destination \"$dir\\Bin\\\"",
+            "Expand-7zipArchive \"$dir\\.rsrc\\2052\\MSI\\102\" \"msvcp100.dll\" -Destination \"$dir\\Bin\\\"",
+            "Expand-7zipArchive \"$dir\\.rsrc\\2052\\MSI\\102\" \"TXSSO\\TXSSOSetup.exe\" -Destination \"$dir\" -Removal",
+            "Expand-7zipArchive \"$dir\\TXSSO\\TXSSOSetup.exe\" \"TXSSO\" -Destination \"$dir\\bin\" -Removal"
+        ]
+    },
+    "post_install": [
+        "Echo \"$dir\\Files\" \"$dir\\.rsrc\" \"$dir\\TXSSO\" | Remove-Item -Recurse"
+    ],
```

Comment:
```suggestion
    "installer": {
        "script": [
            "Get-Item \"$dir\\*\" -Exclude .rsrc | Remove-Item",
            "Expand-7zipArchive \"$dir\\.rsrc\\2052\\MSI\\101\" \"$dir\" -ExtractDir 'Files'",
            "Expand-7zipArchive \"$dir\\.rsrc\\2052\\MSI\\102\" \"$dir\\temp\"",
            "'vcruntime140.dll', 'msvcp140.dll', 'msvcr100.dll', 'vcomp140.dll', 'msvcp100.dll' | ForEach-Object {",
            "  Move-Item \"$dir\\temp\\$_\" \"$dir\\Bin\\\"",
            "}",
            "Expand-7zipArchive \"$dir\\temp\\TXSSO\\TXSSOSetup.exe\" \"$dir\\Bin\\TXSSO\" -ExtractDir 'TXSSO'",
            "Remove-Item \"$dir\\.rsrc\", \"$dir\\temp\" -Recurse"
        ]
    },
```

Simplified `installer.script`, and got same folder structure. Please test it.

![image](https://user-images.githubusercontent.com/5832170/220412489-cba75de5-9023-4705-8947-a4c557d4a1b2.png)